packages/mux-audio/package.json

```diff
@@ -47,11 +47,11 @@
     "publish-release": "../../scripts/publish.sh"
   },
   "dependencies": {
-    "@mux/playback-core": "0.9.0"
+    "@mux/playback-core": "0.9.0",
+    "@mux/polyfills": "0.1.0"
```

Comment:
issue(blocking): Can't be a dependency. polyfills isn't published 💔

Reply:
- made a atmux/polyfills -> shared-polyfills so we can still use that atmux scope.
- since it's bundled in no need to create builds/bundles for shared-polyfills